libclamav/scanners.c

```diff
@@ -4652,47 +4653,76 @@ cl_error_t cli_magic_scan(cli_ctx *ctx, cli_file_t type)
              * Note: JPEG 2000 is a very different format from JPEG, JPEG/JFIF, JPEG/Exif, JPEG/SPIFF (1994, 1997)
              * JPEG 2000 is not handled by cli_scanjpeg or cli_parsejpeg.
              */
-            ret = calculate_fuzzy_image_hash(ctx, type);
+
+            // It's okay if it fails to calculate the fuzzy hash.
+            (void)calculate_fuzzy_image_hash(ctx, type);
```

Comment:
Very minor, but should probably add a space between (void) and function name

Reply:
I don't really have an opinion on this. But I just tested it and our clang-format settings refactor it to remove the space.